charts/falco/templates/pod-template.tpl

```diff
@@ -65,7 +65,7 @@ spec:
         {{- if .enabled }}
         {{- if .containerd.enabled }}
         - --cri
-        - /run/containerd/containerd.sock
+        - /run/containerd/{{ base .containerd.socket }}
```

Comment:
Could you do the same for `docker` and `crio` container runtimes?

Reply:
Done - sorry for the delay :|